scripts/packages/VSCodeServer/src/repl_protocol.jl

```diff
@@ -8,11 +8,18 @@ JSONRPC.@dict_readable struct ReplRunCodeRequestParams <: JSONRPC.Outbound
     showResultInREPL::Bool
 end
 
+struct Frame
+    path::String
+    line::Int
+end
+Frame(st::Base.StackFrame) = Frame(fullpath(string(st.file)), st.line)
+
 JSONRPC.@dict_readable struct ReplRunCodeRequestReturn <: JSONRPC.Outbound
     inline::String
     all::String
-    iserr::Bool
+    stackframe::Union{Nothing,Vector{Frame}}
 end
+ReplRunCodeRequestReturn(inline, all) = ReplRunCodeRequestReturn(inline, all, nothing)
```

Comment:
@davidanthoff sorry for turning back the discussion, but do you think we can change the [custom](https://github.com/julia-vscode/julia-vscode/pull/1371#discussion_r443005108) ?
After writing this code, I honestly feel a bit awkward that I need to define `Frame` here, because `Frame` (and its constructor method) really depends on the logic inside eval.jl, but because I had to define this here, I had to switch between files while writing this code.
If we're okay to split these protocols into each feature file, we can just focus on one single file while developing a feature, and that would feel much better, I guess.